spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1348,6 +1348,25 @@ In the previous example, the `BeanWrapperFieldExtractor` described earlier in th
 chapter is used to turn the name and credit fields within `CustomerCredit` into an object
 array, which is then written out with commas between each field.
 
+It is also possible to use the `FlatFileItemWriterBuilder.DelimitedBuilder` to
+automatically create the `BeanWrapperFieldExtractor` and `DelimitedLineAggregator`
+as shown in the following example:
+
+.Java Configuration
+[source, java, role="javaContent"]
+----
+@Bean
+public FlatFileItemWriter<CustomerCredit> itemWriter(Resource outputResource) throws Exception {
+	return new FlatFileItemWriterBuilder<CustomerCredit>()
+				.name("customerCreditWriter")
+				.resource(outputResource)
+				.delimited()
+				.delimiter(",")
```

Comment:
Setting a coma as a delimiter here doesn't make sense since that's the default.  I can update on merge.

Reply:
make sense.

> I can update on merge.

Yes please. Many thanks!